tensorflow/compiler/xla/service/cpu/ir_emitter.cc

```diff
@@ -1926,8 +1927,173 @@ Status IrEmitter::HandleSendDone(HloInstruction* send_done) {
   return Unimplemented("Send-done is not implemented on CPU.");
 }
 
-Status IrEmitter::HandleScatter(HloInstruction*) {
+Status IrEmitter::HandleScatter(HloInstruction* scatter) {
```

Comment:
This was a tentative of porting the pre-mlir GPU scatter emitter (TF git tag 2.4.3).